<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Consola</class>
 <widget class="QWidget" name="Consola">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="fondoC">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>563</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="salidas">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>500</width>
     <height>250</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QGraphicsView" name="visorImagenes">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>IDE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verImagenButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Adlam</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ver imagen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
